specifyweb/frontend/js_src/lib/components/WbUtils/index.tsx

```diff
@@ -127,6 +129,28 @@ export function WbUtilsComponent({
         totalCount={cellCounts.newCells}
         utils={utils}
       />
+      {
```

Comment:
what is the difference between "Matched and Changed" vs "Updated"?

Reply:
Updated means one of the mapped columns changed. So, collectionobject's remarks field changed -- that'll be updated.

Matched and Changed means if you mapped cataloger, and it matched it to an agent different than the current one. In this case, the collectionobject will have NoChange (completely of no color), but the agent will be of different color (because matched and changed). This was done because otherwise you'd be highlighting the entire collectionobject, which introduces ambiguity: did you actually change all the fields, or just the relationship?

Reply:
ah, makes sense.
vanilla workbench has this ambiguity when you add determinations to existing collection object